src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/AuthorAndDownloadCount.xaml.cs

```diff
@@ -95,7 +95,7 @@ private static void OnPropertyChanged(
 
         public event PropertyChangedEventHandler PropertyChanged;
 
-        private void OnPropertyChanged(string propertyName)
+        protected void OnPropertyChanged(string propertyName)
```

Comment:
I'm pretty sure this whole method can be deleted but it'll be a bigger change. Let me know if you want to do it and we can chat about it.